setup.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="graviton",
+    version="0.0.1",
+    author="Algorand",
+    author_email="pypiservice@algorand.com",
+    description="verify your TEAL program by experiment and observation",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/algorand/graviton",
+    packages=setuptools.find_packages(),
+    install_requires=["py-algorand-sdk", "tabulate==0.8.9"],
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: MIT License",
+        "Operating System :: OS Independent",
+    ],
+    python_requires=">=3.8",
```

Comment:
I'm stuck with this dependency because of our C.I. process which uses ubuntu-20 and on which it seems rather difficult to get python 3.10 going on.

Reply:
@michaeldiamant ^^^ this might be an issue for our goal of getting PyTEAL to 3.10 as well

Reply:
@tzaffi I opened a child PR to help move to 3.10:  https://github.com/algorand/graviton/pull/8.  Can you have a look and let me know if you're up for merging it?

Note:  The build did _not_ find a cached sandbox.  So, I'm leaving the message ahead of the build completing.  Local testing `act` suggests it works.  I'll double check tomorrow.  🙏 

Aside:  Is it possible to configure an upper bound for supported Python versions?
* It's _not_ a pressing concern.  
* My passing thought is:  If the build claims to support newer versions, then the CI build ought to enforce it (e.g. matrix build).  For example, consider Python 3.11 coming out, but not being built by CI.

Reply:
This is nice. I'll work to incorporate this into the PR!

Reply:
FYI: 
* I merged #8  in. Works great!
* I tried naively adding "3.11" to the github-actions python matrix, and it didn't work. It appears [not to yet be supported](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md#python).